pkg/abstractions/image/image.go

```diff
@@ -55,20 +57,23 @@ func NewRuncImageService(
 func (is *RuncImageService) VerifyImageBuild(ctx context.Context, in *pb.VerifyImageBuildRequest) (*pb.VerifyImageBuildResponse, error) {
 	var valid bool = true
 
-	baseImageTag, ok := is.config.ImageService.Runner.Tags[in.PythonVersion]
+	baseImageTag, ok := is.getBaseImageTag(in.PythonVersion, in.Micromamba)
 	if !ok {
+		// FIXME: update to account for micromamba
 		return nil, errors.Errorf("Python version not supportted: %s", in.PythonVersion)
 	}
 
 	opts := &BuildOpts{
-		BaseImageTag:      baseImageTag,
-		BaseImageName:     is.config.ImageService.Runner.BaseImageName,
-		BaseImageRegistry: is.config.ImageService.Runner.BaseImageRegistry,
-		PythonVersion:     in.PythonVersion,
-		PythonPackages:    in.PythonPackages,
-		Commands:          in.Commands,
-		BuildSteps:        convertBuildSteps(in.BuildSteps),
-		ExistingImageUri:  in.ExistingImageUri,
+		BaseImageTag:       baseImageTag,
+		BaseImageName:      is.config.ImageService.Runner.BaseImageName,
+		BaseImageRegistry:  is.config.ImageService.Runner.BaseImageRegistry,
+		PythonVersion:      in.PythonVersion,
+		PythonPackages:     in.PythonPackages,
+		Commands:           in.Commands,
+		BuildSteps:         convertBuildSteps(in.BuildSteps),
+		ExistingImageUri:   in.ExistingImageUri,
+		Micromamba:         in.Micromamba,
```

Comment:
Can we add this to buildsteps instead of adding additional parameters for mamba?

Reply:
I ended up removing these fields and setting the python version to micromamba in the sdk. That can be used in place of this flag. Also, channels can just be added as build steps to be parsed as flags, so I did that as well.